lib/restaurant.rb

```diff
@@ -0,0 +1,20 @@
+require './lib/sendtext'
+
+class Restaurant
+  attr_reader :menu, :sendtext
+
+  def initialize(sendtext = SendText.new)
+    @menu = {"Chips" => 0.89, "Chicken Tikka" => 3.99, "Burger" => 2.99, "Veggie Wrap" => 4.99}
```

Comment:
Metrics/LineLength: Line is too long. [95/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)